tests/fast-integration/utils/index.js

```diff
@@ -483,6 +483,16 @@ async function getAllSubscriptions(namespace = 'default') {
   }
 }
 
+// gets the active eventing backend
+async function getEventingBackend(namespace = 'kyma-system') {
+  const path = '/apis/eventing.kyma-project.io/v1alpha1/eventingbackends/';
+  const response = await k8sDynamicApi.requestPromise({
+    url: k8sDynamicApi.basePath + path,
+  });
+  const body = JSON.parse(response.body);
+  return body?.items[0]?.status?.backendType;
```

Comment:
Unfortunately I couldn't some other way to fetch the backend. Regarding possible 2nd backend I think these tests will fail and we will then try to fix them. So IMO I would go step by step and not try to think about everything.

Reply:
I don't think fetching the backend this way is a problem 😃Are you sure the tests will fail ?

I suggest iterating over the entries in `response.body` and finding the `eventingbackend` with name `eventing-backend`. In case that there are multiple backends, we will choose the one named with `eventing-backend`.
With the current approach we just choose the first, so basically the behaviour is undefined in case of 2 `eventingbackends`.

Reply:
done